bigquery_etl/metadata/parse_metadata.py

```diff
@@ -343,6 +345,9 @@ def from_file(cls, metadata_file):
                         # column needs to be set explicitly
                         monitoring.partition_column_set = True
 
+                if "enforce_col_desc" in metadata:
+                    enforce_col_desc = metadata["enforce_col_desc"]
```

Comment:
Can we rename this to `enforce_column_descriptions`. Generally, we have been using more descriptive names in the metadata files (even though they are a little bit longer)

Reply:
IMO `require_column_descriptions` would be slightly more to the point.